docs/ac-remote-update/how-it-works.md

```diff
@@ -0,0 +1,61 @@
+# Remote update
+
+Remote update is a capability of Agent Control that enable updating its version remotely, using Fleet Control. That includes Flux in Kubernetes. However, this feature is not available in all the supported systems yet.
+
+## Supported systems
+
+| System     | Supported |
+|------------|-----------|
+| OnHost     | ❌        |
+| Kubernetes | ✅        |
+
+### Kubernetes
+
+In Kubernetes, both the Agent Control version and Flux version can be updated. We provide a [Helm Chart to install Agent Control](https://github.com/newrelic/helm-charts/tree/master/charts/agent-control), which installs the latest version of Agent Control and Flux 2.15.0. There are some caveats.
+
+#### Caveats
+
+##### Updating Agent Control version
+
+We could receive a remote configuration with an older Agent Control version which doesn't support this feature. In that situation, the Agent Control would end up in a "non-recoverable" state. Updating it would require access to the system. Ideally, the backend should prevent that.
```

Comment:
There could be also problems with supported agent-types. If the AC version changes to a new version where a specific agent-type is not supported yet (or is not supported anymore) it won't work properly.